Automattic/jetpack
# Update/show crm active on my jetpack

## Proposed changes:

* This diff shows Jetpack CRM as active on My Jetpack if the CRM plugin is installed and active

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a new test site with Jetpack Beta
* Apply this diff and view the My Jetpack page.
* Confirm that the CRM card on My Jetpack shows "Learn More" since the plugin has not been installed yet.
* Click "Learn more" on the CRM card - this will take you to the interstitial page for CRM
* From the interstitial page, click "Install Jetpack CRM"
* After the installation is done, you should be redirected to the main page for the CRM plugin
* Now, navigate back to My Jetpack.
* Confirm that the CRM card now shows as active
![Screenshot 2024-03-26 at 4 28 17 PM](https://github.com/Automattic/jetpack/assets/18016357/b6de7f21-86b7-4775-a411-e83f6f7eb089)
